evap/evaluation/models.py

```diff
@@ -1560,6 +1560,9 @@ def sorting_key(self): # keep in sync with Meta#ordering
             (self.email or "").lower()
         )
 
+    def __lt__(self, other):
+        return self.sorting_key < other.sorting_key
```

Comment:
Django has a standardized way of specifying the default ordering, though [`Meta.ordering`](https://docs.djangoproject.com/en/3.2/ref/models/options/#django.db.models.Options.ordering). Django has explicitly decided _not_ to implement `__lt__()` for the models when it was requested ([ticket 26141](https://code.djangoproject.com/ticket/26141), [ticket 22843](https://code.djangoproject.com/ticket/22843)).

When defining one comparison operator, users of the class would usually expect that you have some kind of total or partial order that is being expressed, and that the remaining comparison operators are defined and behave as [would commonly be expected](https://stackoverflow.com/a/1061350/12345551).
However, we can not achieve a meaningful implementation for all the operators here (see discussion above). Django ships a default implementation for `__eq__()` that uses an object's primary key. Our comparison operators would use some other attributes. I don't see how this could work together semantically.
We would be left with implementing only `__lt__()` for the purpose of sorting objects, but not comparing them otherwise.

Considering that we often have new contributors in this project, I think this would be a source of confusion. In most cases, django's ordering feature for database queries should be usable (also in this case, see https://github.com/e-valuation/EvaP/pull/1646#discussion_r732000220), and if we don't have a real need for this, I would avoid it.